src/diamond/pycompat.py

```diff
@@ -0,0 +1,18 @@
+# coding=utf-8
```

Comment:
We should just start by vendorizing the entirety of six for now imo. There are too many things we're using.